Goutte/Client.php

```diff
@@ -129,14 +129,16 @@ protected function doRequest($request)
                 $requestOptions['body'] = $content;
             } else {
                 if ($files = $request->getFiles()) {
-                    $requestOptions['multipart'] = [];
+                    $requestOptions['multipart'] = array();
```

Comment:
Maybe we can create a new Pull request which takes care of that coding standard changes?